.github/workflows/selenium_beta.yaml

```diff
@@ -11,6 +11,7 @@ on:
     - cron: '0 0 * * 2'
 env:
   GALAXY_CONFIG_GALAXY_URL_PREFIX: '/galaxypf'
+  GALAXY_CONFIG_USE_LEGACY_HISTORY: false
```

Comment:
That's the default and so has no effect, I think you'll want to switch and set this to true in the non-beta selenium variants.

Reply:
Yep, should have swapped this over.

Reply:
https://github.com/galaxyproject/galaxy/pull/13723/commits/d9057ed18dbae3357202743b83258179bd8aa75a -- that also sets the beta variant as `continue-on-error`, allowing the workflow to fail but won't affect the check status.